dev/TEAL_opcodes.md

```diff
@@ -1304,25 +1308,43 @@ bitlen interprets arrays as big-endian integers, unlike setbit/getbit
 - LogicSigVersion >= 5
 - Mode: Application
 
-`tx_begin` initializes Sender to the application address; Fee to the minimum allowable, taking into account MinTxnFee and credit from overpaying in earlier transactions; FirstValid/LastValid to the values in the top-level transaction, and all other fields to zero values.
+`itxn_begin` initializes Sender to the application address; Fee to the minimum allowable, taking into account MinTxnFee and credit from overpaying in earlier transactions; FirstValid/LastValid to the values in the top-level transaction, and all other fields to zero values.
 
-## tx_field f
+## itxn_field f
 
 - Opcode: 0xb2 {uint8 transaction field index}
 - Pops: *... stack*, any
 - Pushes: _None_
-- Set field F of the current inner transaction to X. Fail if X is the wrong type for F.
+- Set field F of the current inner transaction to X
 - LogicSigVersion >= 5
 - Mode: Application
 
-`tx_field` fails if X is of the wrong type for F, including a byte array of the wrong size for use as an address when F is an address field. `tx_field` also fails if X is an account or asset that does not appear in `txn.Accounts` or `txn.ForeignAssets` of the top-level transaction. (Setting addresses in asset creation are exempted from this requirement.)
+`itxn_field` fails if X is of the wrong type for F, including a byte array of the wrong size for use as an address when F is an address field. `itxn_field` also fails if X is an account or asset that does not appear in `txn.Accounts` or `txn.ForeignAssets` of the top-level transaction. (Setting addresses in asset creation are exempted from this requirement.)
 
-## tx_submit
+## itxn_submit
```

Comment:
Another corner case: what happens if you execute twice `itxn_submit` without doing `itxn_begin` in between?

For example, suppose I want to send 42 Algos to two different receivers can I do?

```
itxn_begin
// setup everything with itxn_field

addr FIRST_RECEIVER
itxn_field Receiver

itxn_submit

addr SECOND_RECEIVER
itxn_field Receiver

itxn_submit
```

note that I did not set any other field for the second transaction.

And that comes back to the question of "unsetting" a field.
In that setting it makes sense to "unset" a field such as `CloseTo` or `Amount` to switch to asset transfer.

Reply:
documented.